### PR TITLE
Run ii 106 x v1

### PR DESCRIPTION
**Pull request by anmalara**
Cherry-picked modification for JER from 102X_v1
Removed old dependancies for JER and JEC.
In the next version of the code we will use only central tools and therefore the old declaration is outdated.

### CODE REVIEW

**Comment by anmalara** [comment]
@raggleton Can you merge this PR and do a cherry-pick of the 
93b16a0 commit to the other branches? Thanks

File: scripts/crab/readaMCatNloEntries.py [hunk 1]
```diff
@@ -28,15 +28,15 @@ def read_tree(rootDir):
     numberOfweightedEntries = 0
     try:
         # Use C++ script to count as significantly faster (~15x)
-        cmd = "root -q -b -l 'countNumberEvents.C+(\""+rootDir+"\",false)'"
+        cmd = "root -q -b -l 'countNumberEvents.C(\""+rootDir+"\",false)'"
```
Comment (raggleton) [inline scripts/crab/readaMCatNloEntries.py]: The + compiles the macro (bit better than just running it as a macro)

**Comment by anmalara** [comment]
I removed because it was not strictly necessary and when running the macro with multicrab it leaves many compilation files. I thought could be annoying but if you want I can put it back.

**Comment by anmalara** [comment]
Done. Please check and merge.

File: scripts/crab/readaMCatNloEntries.py [hunk 2]
```diff
@@ -76,7 +78,9 @@ def readEntries(worker, xmlfiles, fast=False):
         pool.join()
         entries_per_rootfile = [r for r in result.get()]
         print "number of events in",xml,sum(entries_per_rootfile)
+	result_list.append(sum(entries_per_rootfile))
```
Comment (raggleton) [inline scripts/crab/readaMCatNloEntries.py]: Can you fix the spacing here please? Looks like tab & spaces got mixed up - on github the indentation looks weird

**Pull request merged**